app/lib/features/chat/widgets/rooms_list.dart

```diff
@@ -128,35 +140,13 @@ class _RoomsListWidgetState extends ConsumerState<RoomsListWidget> {
             ref.read(roomListFilterProvider.notifier).updateSearchTerm(value);
           },
         ),
-        const SizedBox(height: 20),
-        SegmentedButton<FilterSelection>(
-          segments: const [
-            ButtonSegment(
-              value: FilterSelection.all,
-              label: Text('all'),
-            ),
-            ButtonSegment(
-              value: FilterSelection.favorites,
-              icon: Icon(Icons.bookmark_outline),
-            ),
-            ButtonSegment(
-              value: FilterSelection.dmsOnly,
-              label: Text("only DM's"),
-            ),
-          ],
-          onSelectionChanged: (Set<FilterSelection> newSelection) async {
-            await ref
-                .read(roomListFilterProvider.notifier)
-                .setSelection(newSelection.first);
-          },
-          selected: {selected},
-        ),
-        const SizedBox(height: 10),
+        filterChipsButtons(),
         Row(
           mainAxisAlignment: MainAxisAlignment.end,
           children: [
+            Expanded(child: searchTerms(context)),
```

Comment:
I understand correctly you removed the search term from the title and only show it while the box is visible now? What if you scrolled down and the search box disappears? The information of it being a search also disappears then? 

If we have a search term that we filter by that information must be sticky on screen or we'll see a bunch of "my chat's disappeared"-bug-reports of people forgetting that they had searched for things (the search info disappearing due to it not being synced back from the provider to the UI if the user left the UI was already an annoying bug we had in the past).

This information must be shown regardless of whether the filter-box is visible or not.

Reply:
> If we have a search term that we filter by that information must be sticky on screen or we'll see a bunch of "my chat's disappeared"-bug-reports of people forgetting that they had searched for things (the search info disappearing due to it not being synced back from the provider to the UI if the user left the UI was already an annoying bug we had in the past).

Search terms doesn't show and cutting off as shown in below screenshot. We can make it for two line but again I feel that title bar should not be like that.

![Screenshot 2024-03-06 at 5 59 45 PM](https://github.com/acterglobal/a3/assets/51526480/de422089-7a5a-46e0-964f-6420f8a55cfa)


> I understand correctly you removed the search term from the title and only show it while the box is visible now? What if you scrolled down and the search box disappears? The information of it being a search also disappears then?
This information must be shown regardless of whether the filter-box is visible or not.

Let me think on it and do the needful.